airbyte-webapp/src/packages/cloud/views/layout/SideBar/SideBar.tsx

```diff
@@ -22,16 +21,12 @@ import OnboardingIcon from "views/layout/SideBar/components/OnboardingIcon";
 import SettingsIcon from "views/layout/SideBar/components/SettingsIcon";
 import SidebarPopout, { Icon, Item } from "views/layout/SideBar/components/SidebarPopout";
 import SourceIcon from "views/layout/SideBar/components/SourceIcon";
+import { CreditsIcon } from "../../../../../components/icons/CreditsIcon";
```

Comment:
This suggestion should be applied locally so that the linter can sort the imports correctly.
```suggestion
import { CreditsIcon } from "components/icons/CreditsIcon";
```